scan

```diff
@@ -1,8 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
This change seems unrelated?

Reply:
Yes. This is a dysfunctional script that was left over after refactoring of the general scan command line script.